doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -267,35 +306,35 @@ Simulation in CLI
 
 .. code-block:: bash
 
-    # Flower 1.8
+    # Flower v1.8
```

Comment:
```suggestion
    # Flower v1.13+
    $ flwr run
    
    # Flower v1.8 - v1.10
```

@jafermarq I'm not familiar with the `flower-simulation` versions. Is this part correct?